### PR TITLE
chore(linux): Check and restart background processes

### PR DESCRIPTION
**Pull request by ermshiperete**
Before we run a test we now check that ibus and the other processes we need are still running. If not we run test setup again.

This will likely fix #9570.

@keymanapp-test-bot skip

### CODE REVIEW

File: linux/ibus-keyman/tests/scripts/run-single-test.sh [hunk 1]
```diff
@@ -34,6 +34,8 @@ function help() {
   echo "  --wayland               run tests with Wayland"
   echo "  --x11                   run tests with X11"
   echo "  --env <envfile>         Name of the file containing environment variables to use"
+  echo "  --check <pidfile>       Name of the file containing pids to check are running"
+  echo "  --cleanup <file>        Name of the file containing cleanup of processes"
```
Comment (darcywong00) [inline linux/ibus-keyman/tests/scripts/run-single-test.sh:38]: Does the usage on l.24 get longer with these additional args? :)

File: linux/ibus-keyman/tests/scripts/test-helper.inc.sh [hunk 2]
```diff
@@ -176,25 +181,31 @@ function _setup_display_server() {
...
     echo "export DISPLAY=\"$DISPLAY\"" >> "$ENV_FILE"
   fi
 }
 
 function _setup_schema_and_gsettings() {
-  local ENV_FILE PID_FILE
+  local ENV_FILE CLEANUP_FILE
   ENV_FILE=$1
-  PID_FILE=$2
+  CLEANUP_FILE=$2
```
Comment (darcywong00) [inline linux/ibus-keyman/tests/scripts/test-helper.inc.sh:208]: CLEANUP_FILE looks unused. Maybe it can be removed?

**Review by darcywong00 — APPROVED** [review]
lgtm

File: linux/ibus-keyman/tests/scripts/run-single-test.sh (hunk 1 above)
Comment (ermshiperete) [reply linux/ibus-keyman/tests/scripts/run-single-test.sh:38]: Thanks!

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.181-alpha](https://keyman.com/downloads/releases/alpha/17.0.181)

**Pull request merged**